tests/test_controller.py

```diff
@@ -33,10 +33,10 @@ def test_registry(controller, decentralizedController):
     with raises(TransactionFailed): controller.assertOnlySpecifiedCaller(tester.a2, key2, sender = tester.k2)
 
 def test_suicide(controller, decentralizedController, controllerUser):
-    with raises(TransactionFailed): controller.suicide(controllerUser.address, tester.a0, sender = tester.k2)
+    with raises(TransactionFailed): controller.suicide(controllerUser.address, tester.a0, [], sender = tester.k2)
```

Comment:
Recommend adding a test that includes a token (the test legacy rep token is probably easiest as you can just give yourself some REP, transfer it to the controlled contract, then suicide with that address in the array.